app/assets/javascripts/account/AccountService.js.coffee

```diff
@@ -168,15 +168,13 @@ AccountService = (
     )
 
   Service.updateAccount = (infoType) ->
-    bsLoadingOverlayService.start()
     # have to later manually call overlay.stop() since this update doesn't result in a stateChange
     Service.clearAccountMessages()
     if infoType == 'email'
       params =
         user:
           email: Service.userAuth.user.email
       $http.put('/api/v1/auth', params).success((data) ->
-        bsLoadingOverlayService.stop()
         Service.accountSuccess.messages.email = $translate.instant("ACCOUNT_SETTINGS.VERIFY_EMAIL")
       ).error((response) ->
         bsLoadingOverlayService.stop()
```

Comment:
@chrisdolendo Do we still need to call `bsLoadingOverlayService.stop()` here since the call to `bsLoadingOverlayService.start()` was removed? Also then maybe we can remove the comment above, `# have to later manually call overlay.stop() since this update doesn't result in a stateChange`.

Reply:
good catch @callaghanc, I meant to remove both. Will remove that line and the comment.